common/src/IO/MapReader.cpp

```diff
@@ -404,52 +407,20 @@ namespace TrenchBroom {
             }
         }
 
-        /**
-         * Transforms m_brushInfos into a vector of LoadedBrush (leaving m_brushInfos empty).
-         */
-        std::vector<MapReader::LoadedBrush> MapReader::loadBrushes(ParserStatus& /* status */) {
-            // In parallel, create Brush objects (moving faces out of m_brushInfos)
-            auto loadedBrushes = kdl::vec_parallel_transform(std::move(m_brushInfos), [&](BrushInfo&& brushInfo) {
-                LoadedBrush result;
-                result.brush = std::make_optional(Model::Brush::create(m_worldBounds, std::move(brushInfo.faces)));
-                result.startLine = brushInfo.startLine;
-                result.lineCount = brushInfo.lineCount;
-                return result;
-            });
-
-            assert(m_brushInfos.empty());
-
-            return loadedBrushes;
-        }
-
         Model::Node* MapReader::resolveParent(const ParentInfo& parentInfo) const {
-            if (parentInfo.layer()) {
-                const Model::IdType layerId = parentInfo.id();
+            if (parentInfo.type == ParentType::Layer) {
+                const Model::IdType layerId = parentInfo.id;
                 const auto it = m_layers.find(layerId);
                 return it != std::end(m_layers) ? it->second : nullptr;
+            } else {
+                const Model::IdType groupId = parentInfo.id;
+                const auto it = m_groups.find(groupId);
+                return it != std::end(m_groups) ? it->second : nullptr;
             }
-            const Model::IdType groupId = parentInfo.id();
-            const auto it = m_groups.find(groupId);
-            return it != std::end(m_groups) ? it->second : nullptr;
-        }
-
-        MapReader::EntityType MapReader::entityType(const std::vector<Model::EntityProperty>& properties) const {
-            const std::string& classname = findProperty(properties, Model::PropertyKeys::Classname);
-            if (isLayer(classname, properties))
-                return EntityType_Layer;
-            if (isGroup(classname, properties))
-                return EntityType_Group;
-            if (isWorldspawn(classname, properties))
-                return EntityType_Worldspawn;
-            return EntityType_Default;
-        }
-
-        void MapReader::setFilePosition(Model::Node* node, const size_t startLine, const size_t lineCount) {
-            node->setFilePosition(startLine, lineCount);
         }
 
         /**
-         * Default implementation adds it to the current BrushInfo
+         * Default implementation adds it to the current detail::BrushInfo
```

Comment:
this change is obsolete, I don't see a `detail` namespace?

Reply:
fixed